civihr_employee_portal/features/civihr_employee_portal_features/civihr_employee_portal_features.features.filter.inc

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * @file
+ * civihr_employee_portal_features.features.filter.inc
+ */
+
+/**
+ * Implements hook_filter_default_formats().
+ */
+function civihr_employee_portal_features_filter_default_formats() {
+  $formats = array();
```

Comment:
please use the `[]` notation (applies everywhere in this PR

Reply:
This is auto-generated code from features. I think it is using this syntax to maximize compatibility.